api/Dockerfile

```diff
@@ -13,7 +13,13 @@ RUN apt-get -y install unixodbc-dev
 # Install old (2.4.*; current debian) version of gdal
 RUN apt-get -y install libgdal-dev
 
-# Update pip  
+# Install R
```

Comment:
Is this docker file intended to just be build and used locally or is it part of the openshift image build pipeline? If the latter are we installed R twice? It also gets installed in openshift/wps-api-base/docker/Dockerfile

Reply:
You'll see the same command in 3 docker files - but for any one image, it's only installed once.
1) api/Dockerfile - this docker file is for local development only
2) openshift/ubuntu-base/docker/Dockerfile - this docker file only used for the c-haines cronjob (needs ubuntu for most recent gdal)
3) openshift/wps-api-base/docker/Dockerfile - this is the base image that all our PR's use. It's the fastapi docker image with our magic juice added on top.